experiment-reason/src/lib/Governance.rei

```diff
@@ -0,0 +1,24 @@
+/* Person to be proposed as new member for the project. */
+type candidate = string
+
+// Amount of shares granted for the new member.
+type shares = int
+
+// Amount of coins to be payed by the candidate in exchange for shares in the
```

Comment:
pay /peɪ/ (verb NAUTICAL) past tense: payed - seal (the deck or seams of a wooden ship) with pitch or tar to prevent leakage "an open groove between the planks had to be payed by running in hot pitch from a special ladle"

you probably meant "paid" ; P

Reply:
Hahaha, ze german coming through.